qa/suites/upgrade/pacific-p2p/pacific-p2p-stress-split/6-final-workload/rbd-python.yaml

```diff
@@ -6,5 +6,5 @@ tasks:
     tag: v16.2.7
     clients:
       client.0:
-        - rbd/test_librbd_python.sh
+        - rbd/test_librbd_python.sh -k "not test_diff_iterate"
```

Comment:
This doesn't work:
- `test_librbd_python.sh` doesn't propagate its arguments to `nose`
- Separately, I don't think `nose` has `-k` option.  Can you point at where it's documented?